source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -274,8 +274,9 @@ package:
 
    * - ``BsonCreator``
      - Marks a public constructor or a public static method as the creator for
-       new instances of the class. You can combine this with the
-       ``BsonProperty`` annotation to link the parameters with properties.
+       new instances of the class. All parameters in the constructor must
+       be annotated with a ``BsonProperty``. You must use the annotation
```

Comment:
Issue: I think this contains passive voice, consider revising.

```suggestion
       new instances of the class. All parameters in the constructor must
       have a ``BsonProperty`` annotation. You must use the annotation
```

Reply:
I'll reword to be more specific, not sure the suggestion provides enough direction for the reader.